lib/linguist/languages.yml

```diff
@@ -5264,6 +5264,15 @@ UrWeb:
   tm_scope: source.ur
   ace_mode: text
   language_id: 383
+V:
+  type: programming
+  color: "#00ADD8"
+  extensions:
+  - ".v"
+  ace_mode: golang
+  codemirror_mode: go
+  codemirror_mime_type: text/x-go
+  language_id: 132
```

Comment:
Please follow the "Adding a Language" instructions.  You've clearly _not_ 😉

Reply:
where do i get new language_id? or it is just random number?

Reply:
found the update-ids script and ran it, and updated PR.